source/fundamentals/connection/connect.txt

```diff
@@ -55,27 +55,48 @@ A standard connection string includes the following pieces:
        :ref:`csharp-connection-options` for a full description of
        these options.
 
+To use a connection URI, pass it as a string to the ``MongoClient`` constructor. In the 
+following example, the driver uses a sample connection URI to connect to a MongoDB 
+instance on port ``27017`` of ``localhost``: 
+
+.. literalinclude:: /includes/fundamentals/code-examples/connection/LocalConnection.cs
+   :language: csharp
+
 .. tip::
 
     See :manual:`the MongoDB Manual</reference/connection-string>`: for more information about creating a connection string.
 
-Connection Targets
-------------------
+MongoClientSettings
+-------------------
+
+You can use a ``MongoClientSettings`` object to configure the connection in code
+rather than in a connection URI. To use a ``MongoClientSettings`` object, create an 
+instance of the class and pass it as an argument to the ``MongoClient`` constructor. 
+
+In the following example, the driver uses a ``MongoClientSettings`` object to connect to a 
+MongoDB instance on port ``27017`` of ``localhost``:
```

Comment:
Should be fixed.

I thought hard about how to present time spans. My reasoning was that the URI parameter includes "MS" as an indication that it should be milliseconds, and many of the examples on this page use numbers in the thousands to reinforce that idea. I settled on making the default value "X minutes" or "X seconds" to avoid confusion from leaving out the unit altogether.